src/app/Translator.js

```diff
@@ -3,14 +3,21 @@ import {connect} from 'react-redux'
 import {IntlProvider, addLocaleData} from 'react-intl';
 import en from 'react-intl/locale-data/en';
 import es from 'react-intl/locale-data/es';
+import ru from 'react-intl/locale-data/ru';
 import {DEFAULT_LANGUAGE} from 'app/client_config';
 import tt from 'counterpart';
 
-addLocaleData([...en, ...es]);
+addLocaleData([...en, ...es, ...ru]);
 
+tt.registerTranslations('en', require('counterpart/locales/en'));
 tt.registerTranslations('en', require('app/locales/en.json'));
 tt.registerTranslations('es', require('app/locales/es.json'));
```

Comment:
why isn't there a `counterpart/locales/es` call?

Reply:
es is not fully supported yet, I'll will add some more language specific stuff in a separate brunch after this one is merged